toonz/sources/toonz/penciltestpopup.cpp

```diff
@@ -1342,19 +1343,22 @@ PencilTestPopup::PencilTestPopup()
   m_captureButton->setIconSize(QSize(30, 30));
 
   m_captureFilterSettingsBtn->setObjectName("GearButton");
-  m_captureFilterSettingsBtn->setFixedSize(23, 23);
-  m_captureFilterSettingsBtn->setIconSize(QSize(15, 15));
+  m_captureFilterSettingsBtn->setFixedSize(24, 24);
+  m_captureFilterSettingsBtn->setIconSize(QSize(16, 16));
+  m_captureFilterSettingsBtn->setIcon(createQIcon("gear");
```

Comment:
In this line the closing bracket seems to be missing.

Reply:
Thanks for catching that!